LandSandBoat/server
# Add in item_equipment.sql for main hand or no MID items.

Still need to find MIDs for all the other items missing before I start adding (Not currently included in this PR, for a future PR). Added some of the missing item mods, but I want to limit the amount of changes in the submission, so I will add more mods and latents to the next PR.



**_I affirm:_**

## What does this pull request do?



Adds missing items and some of their latents/mods/etc, to item_equipment and item_weapon

## Steps to test these changes



Update the SQL with the tables, use gm commands to additem, see if items added to equipment can be equipped, items added to weapon can we weaponed, and items added to mods can mod stats.
